lib/internal/test_runner/tap_stream.js

```diff
@@ -15,7 +15,7 @@ const {
   RegExpPrototypeSymbolReplace
 } = require('#internal/per_context/primordials')
 const { inspectWithNoCustomRetry } = require('#internal/errors')
-const Readable = require('#internal/streams/readable')
+const { Readable } = require('readable-stream')
```

Comment:
It might be better to keep the `require('#internal/streams/readable')` here and have the internal module export from `readable-stream`. That way we would keep the source as close to the Node.js implementation as we can.

Reply:
the source is not really close to node core mainly due to the major differences in the linter settings.
also `primoadials` in core changed to be a global variable, and in this repo, we still require a file - to avoid using a global variable
So I don't really think it matters that much if we require `readable-stream` directly or not

Reply:
`primordials` is not a global variable in core, it's one of the arguments passed to the CJS-like wrapper function. But point taken, if you think it's better this way, I don't see a problem with it.